examples/python/helloworld/greeter_server.py

```diff
@@ -24,6 +24,7 @@
 class Greeter(helloworld_pb2_grpc.GreeterServicer):
 
     def SayHello(self, request, context):
+        print("Received:", request.name)
```

Comment:
Can we update the print message to make sure client and server print messages are similar? For examle "Greeter server received: ....".